src/Payment.cs

```diff
@@ -62,7 +62,14 @@ public Transfer Transfer(Dictionary<string, object> data = null)
         {
             string relativeUrl = string.Format("{0}/{1}/transfers", GetEntityUrl(), this["id"]);
             List<Entity> entities = Request(relativeUrl, HttpMethod.Post, data);
-            return (Transfer)entities[0];
+
+            // Todo: Make a utility method or something to DRY below for-each thing from across this sdk.
+            List<Transfer> transfers = new List<Transfer>();
+            foreach(Entity entity in entities)
+            {
+                transfers.Add(entity as Transfer);
+            }
+            return transfers;

```

Comment:
you need to change the return type of the function as well.
public List<Transfer> Transfer

Now this seems like a breaking change ? what is someone is using this function and starts getting different return type.

Reply:
Will be a major version bump.

Reply:
Ooops. Did the change.

> Now this seems like a breaking change ? what is someone is using this function and starts getting different return type.

Yes this is a breaking change. And hence would be major release (semver wise).